src/DataFrames.jl

```diff
@@ -128,4 +128,18 @@ for (dir, filename) in [
     include(joinpath(dir, filename))
 end
 
+# Import  "StringEncodings" if available and set up decoder
+_decoder=nothing
+ENCODING_PACKAGE_READY = false
+function __init__()
+    try Pkg.installed("StringEncodings")
```

Comment:
Thanks @tkelman I tested on ` 0.6.0-rc3.0` and it worked fine. 

I will be reverting all the optional loading functionality hacks. Hopeful we can fix that when there is first class support for it at some point.